_extras/glossary.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Glossary 
+---
+
+## ESMValTool Glossary
+
+Here, there are some definitions used throughout this tutorial:
+
+- **Diagnostics**:
+
+- **CMIP**: The Coupled Model Intercomparison Project (CMIP) aims at better understanding past, present and future climate changes arising from natural, unforced variability or in response to changes in radiative forcing in a multi-model context. For more information, check the [WCR-Climate webpage](https://www.wcrp-climate.org/).
+
+- **CMOR**: CMOR stands for [“Climate Model Output Rewriter” library](https://pcmdi.github.io/cmor-site/index.html). It comprises a set of C-based functions, with bindings to both Python and FORTRAN 90, that can be used to produce CF-compliant netCDF files that fulfill the requirements of many of the climate community's standard model experiments.
```

Comment:
In the case of ESMValTool it is more about the formatting rules defined in CMOR standard that the input files must adhere to. And the description of the variables in CMOR can be used as a reference. I think that should be part of the glossary.